src/main/resources/application.yml

```diff
@@ -22,3 +22,10 @@ application:
   version: 1.0
   title: "AYS"
   licenseName: AYS Licence
+
+
+afet:
+  app:
+    secret: afet-secret
+  expires:
+    in: 120000 # 2 minutes
```

Comment:
We can write this as follows:

```
ays:
   token:
      secret: afet-secret
      expires-in: 120000  # 2 minutes
```

Reply:
I just revised the code snippet as you said